components/FilterDropdownSingle/styles.ts

```diff
@@ -13,3 +15,60 @@ export const FilterDropdownInput = styled.select<{ $hasValue: boolean }>`
     background-color: ${COLORS.lightgray};
   }
 `;
+
+// custom styles for react-select component
+// Option type is DropdownOption<T>
+export const customSelectStyles = <T>(
+  $isSmall: boolean,
+): StylesConfig<DropdownOption<T>, true> => ({
+  // container
+  control: (baseStyles, state) => ({
+    ...baseStyles,
+    borderRadius: '57px',
+    border: `0.5px solid ${state.hasValue ? COLORS.shrub : COLORS.midgray}`,
+    backgroundColor: state.isDisabled
+      ? COLORS.lightgray
+      : state.hasValue
+        ? COLORS.shrub
+        : '#fff',
+    padding: '8px 14px',
+    color: COLORS.midgray,
+    minWidth: $isSmall ? '93px' : '150px',
+  }),
+  // placeholder text
+  placeholder: baseStyles => ({
+    ...baseStyles,
+    color: COLORS.midgray,
+    fontSize: '0.75rem',
+    padding: '0px',
+    margin: '0px',
+    justifySelf: 'center',
+  }),
+  // hide vertical bar between arrow and text
+  indicatorSeparator: baseStyles => ({
+    ...baseStyles,
+    display: 'none',
+  }),
+  // 'x' to clear selected option(s)
+  clearIndicator: baseStyles => ({
+    ...baseStyles,
+    padding: '0px',
+  }),
+  // dropdown arrow
+  dropdownIndicator: (baseStyles, state) => ({
+    ...baseStyles,
+    padding: '0px',
+    marginLeft: '-4px', // move the dropdown indicator to the left, cant override text styles
+    color: state.hasValue ? '#fff' : COLORS.midgray,
+  }),
+  // selected option display text
+  singleValue: (baseStyles, state) => ({
+    ...baseStyles,
+    border: '0px',
+    padding: '0px',
+    margin: '0px',
+    fontSize: '0.75rem',
+    color: state.hasValue ? `#fff` : `${COLORS.black} !important`,
```

Comment:
note: my understanding is that `singleValue` will only be rendered if a value is selected, so this check is kinda redundant (i.e. `state.hasValue` will always be true, so we can just directly assign the text color to be white)